src/test/java/dataloader/CsvLoaderTest.java

```diff
@@ -0,0 +1,28 @@
+package dataloader;
+
+import org.junit.Test;
+import tech.tablesaw.api.Table;
+
+import java.io.IOException;
+import java.util.TreeMap;
+
+public class CsvLoaderTest {
+    /**
+     * Test Scenario: The application wants to read a csv dataset
+     * but the String is invalid(empty String).
+     * Expected Output: csvDataLoader throws an IOException.
+     */
+
+    @Test(expected = IOException.class)
+    public void testCsvFileEmptyStringForTree() throws IOException {
+        CsvDataLoader loader = new CsvDataLoader();
+        loader.parseFile("nonExistent.csv", new TreeMap());
+    }
+
+    @Test(expected = IOException.class)
+    public void testCsvFileEmptyStringForTable() throws IOException {
+        CsvDataLoader loader = new CsvDataLoader();
+        Table dataSet = null;
```

Comment:
This test doesn't do what you expect...it throws ```NullPointerException````. Supply a valid ```Table``` instance.

Reply:
pls provide an example & check comment above.

Reply:
check my relevant comment above